CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 22.30.0 - 2023-08-03
+* [#1620](https://github.com/stripe/stripe-java/pull/1620) Update generated code
```

Comment:
Why did this change?

Reply:
merging master?

Reply:
That was a github bug. I didn't pull up updates for sdk-release/next-major. Flipped the base to master and back to `sdk-release/next-major` and things are good now.